roles/monitoring_server/tasks/install_alertmanager_container.yml

```diff
@@ -21,11 +21,7 @@
     image: "{{ monitoring_server_alertmanager.image }}"
     state: started
     user: "{{ monitoring_server_uid }}:{{ monitoring_server_gid }}"
-    command: >
```

Comment:
Was this not working with the line wrapping?

Reply:
Nope - I thought I got it right but I kept getting a 404 from https://mirsg-linux.cs.ucl.ac.uk/alertmanager until I changed it.

Reply:
Weird, we had a similar issue on the SRR XNAT project that Paddy resolved with double quotes around the entire content after the `>` block style indicator. Not sure why, but may be similar here.

Reply:
I'm not sure but as it's a for loop you may need to do `command: >-` to avoid newlines after each iteration and keep it as one command

Reply:
Chomping didn't work and I also tried removing whitespaces in the for loop with `%-` but that didn't work either so I ended splitting over two lines without a yaml block and using a `\`.

Reply:
Frustrating! Probably something subtle happening between the YAML and the templating. Would be nice to figure it out, but don't think it should stop this PR. I'll review the latest changes.